config/.gitignore

```diff
@@ -1,2 +1 @@
 !example.yaml
-!custom-environment-variables.yaml
```

Comment:
We should add this file to .dockerignore too.

Reply:
Good catch.